mrp_production_location_picking_type/models/mrp_production.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    @api.depends("product_id", "company_id", "picking_type_id")
+    def _compute_production_location(self):
+        super()._compute_production_location()
+        for production in self:
+            if (
+                production.picking_type_id
+                and production.picking_type_id.production_location_id
+            ):
+                production.production_location_id = (
+                    production.picking_type_id.production_location_id
+                )
+        return True
```

Comment:
```suggestion
        res = super()._compute_production_location()
        for production in self:
            if (
                production.picking_type_id
                and production.picking_type_id.production_location_id
            ):
                production.production_location_id = (
                    production.picking_type_id.production_location_id
                )
        return res
```
minor suggestion, in upper versions the CI will require super() calls to keep the result